### ISSUE TITLE
SCIE_BOOT=update ./scie-pants fails to verify

### ISSUE THREAD

**Issue by sureshjoshi** [pantsbuild/scie-pants#38]
Just confirming something - does the `SCIE_BOOT=update` require `scie-pants` to be in the PATH? As in, relative calls will fail?

```bash
2022-12-19 21:08:24,219 WARNING] root: Failed to verify scie-pants 0.1.10 installation at scie-pants.
2022-12-19 21:08:24,220 WARNING] root: A backup is saved in scie-pants.bak
2022-12-19 21:08:24,220 ERROR] root: Install failed: [Errno 2] No such file or directory: 'scie-pants'
More information can be found in the log at: /Users/sj/Library/Caches/nce/0fecad25b41fff88ed08979633bf4cc13694d4174b95f5c7e440df42b2376ce5/bindings/logs/update.log
Traceback (most recent call last):
  File "/Users/sj/Library/Caches/nce/0fecad25b41fff88ed08979633bf4cc13694d4174b95f5c7e440df42b2376ce5/bindings/pex_root/venvs/167e2437dc53c975874cbb39299ab31daf082203/5d1a3fc489fcff0ae38dbae6dad477a8ddc54996/pex", line 258, in <module>
    sys.exit(func())
  File "/Users/sj/Library/Caches/nce/0fecad25b41fff88ed08979633bf4cc13694d4174b95f5c7e440df42b2376ce5/bindings/pex_root/venvs/167e2437dc53c975874cbb39299ab31daf082203/5d1a3fc489fcff0ae38dbae6dad477a8ddc54996/lib/python3.9/site-packages/conscript/main.py", line 105, in main
    return ep.load()()
  File "/Users/sj/Library/Caches/nce/0fecad25b41fff88ed08979633bf4cc13694d4174b95f5c7e440df42b2376ce5/bindings/pex_root/venvs/167e2437dc53c975874cbb39299ab31daf082203/5d1a3fc489fcff0ae38dbae6dad477a8ddc54996/lib/python3.9/site-packages/scie_pants/update_scie_pants.py", line 266, in main
    version = verify_release(scie)
  File "/Users/sj/Library/Caches/nce/0fecad25b41fff88ed08979633bf4cc13694d4174b95f5c7e440df42b2376ce5/bindings/pex_root/venvs/167e2437dc53c975874cbb39299ab31daf082203/5d1a3fc489fcff0ae38dbae6dad477a8ddc54996/lib/python3.9/site-packages/scie_pants/update_scie_pants.py", line 184, in verify_release
    subprocess.run(
  File "/Users/sj/Library/Caches/nce/f2bcade6fc976c472f18f2b3204d67202d43ae55cf6f9e670f95e488f780da08/cpython-3.9.15+20221106-x86_64-apple-darwin-install_only.tar.gz/python/lib/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/Users/sj/Library/Caches/nce/f2bcade6fc976c472f18f2b3204d67202d43ae55cf6f9e670f95e488f780da08/cpython-3.9.15+20221106-x86_64-apple-darwin-install_only.tar.gz/python/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/Users/sj/Library/Caches/nce/f2bcade6fc976c472f18f2b3204d67202d43ae55cf6f9e670f95e488f780da08/cpython-3.9.15+20221106-x86_64-apple-darwin-install_only.tar.gz/python/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'scie-pants'
```

Note that I installed 0.1.9, but it appears to have worked regardless:

```bash
⏺ Developer/scratch % PANTS_BOOTSTRAP_VERSION=report ./scie-pants
0.1.10
```

**Comment by jsirois** [comment]
Yup, the fix for #28 fixes this too.

**Comment by jsirois** [comment]
Ok @sureshjoshi 0.1.11 is now live and should fix relative-path use / update. You can also now name the thing `pants` if you want.
